.github/workflows/docker-arm-build-and-test.yaml

```diff
@@ -0,0 +1,74 @@
+name: Regression ARM
+on:
+  push:
+    branches:
+      - arm_test
+jobs:
+  regress_arm:
+    name: ARM PG${{ matrix.pg }} ${{ matrix.build_type }}
+    runs-on: 'ubuntu-18.04'
+    strategy:
+      matrix:
+        pg: [ 12 ]
+        build_type: [ Debug ]
+
+      fail-fast: true
+    env:
+      PG_SRC_DIR: pgbuild
+      PG_INSTALL_DIR: postgresql
+      MAKE_JOBS: 6
+      CLANG: clang-9
+      CC: gcc
+      CXX: g++
+
+    steps:
+    - run: docker run --rm --privileged multiarch/qemu-user-static:register --reset
+
+    - uses: actions/checkout@v1
+
+    - name: Build & Test
+      uses: docker://multiarch/ubuntu-core:arm64-focal
+      with:
+        args: >
+          sh -c "
+            apt update &&
+            apt install -y libssl-dev make postgresql-common gnupg &&
+            yes | /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh &&
+            DEBIAN_FRONTEND=noninteractive apt install -y postgresql-${{ matrix.pg }} postgresql-server-dev-${{ matrix.pg }} cmake sudo libkrb5-dev
+            ./bootstrap -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DREGRESS_CHECKS=OFF &&
+            make -j $MAKE_JOBS -C build &&
+            make -C build install &&
+            chown -R postgres:postgres . &&
+            sudo -u postgres make -C build -k installcheck
+          "
+
+    - name: Show regression diffs
```

Comment:
I guess we are not going to show backtraces here as well?

Reply:
i did get an error with coredumpctl so i didnt include it in initial PR but will investigate if its easy to add